strongbox-data-service/src/main/java/org/carlspring/strongbox/data/service/StrongboxUserService.java

```diff
@@ -0,0 +1,19 @@
+package org.carlspring.strongbox.data.service;
+
+import org.carlspring.strongbox.data.domain.StrongboxUser;
+
+import java.util.Optional;
+
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * @author Alex Oreshkevich
+ */
+@Transactional
+public interface StrongboxUserService
```

Comment:
@AlexOreshkevich:

Could you please rename this to just `UserService` (and the respective implementation as well)?